_sass/_hackathon.scss

```diff
@@ -0,0 +1,33 @@
+
+#csh-hacks-content{
+  padding-top: 10px;
+  padding-bottom: 10px;
+  background: rgba(0,0,0,0.0);
```

Comment:
What's the point of this black background? I don't see that in your mockup.

Reply:
your right, I don't actually need it. Just removed it